nat-lab/docker-compose.yml

```diff
@@ -51,6 +51,8 @@ services:
     cap_drop:
       - ALL
     cap_add:
+      - CAP_DAC_OVERRIDE
+      - CAP_CHOWN
```

Comment:
These two are used in order to make tcpdump work, as else it will fail to create packet capture files